localization/languages/fr-FR.json

```diff
@@ -89,7 +93,7 @@
         "addressInvalidTitle": "Cette adresse est invalide.",
         "genericError": "Il y a une erreur",
         /* pages/phishing/index.html */
-        "phishingErrorTitle": "Ce site peut vous nuire.",
+        "phishingErrorTitle": "Ce site peut vous nuire. (Tentative de Phishing)",
```

Comment:
This looks like it doesn't match the English string, if the title's confusing we should probably try to fix it for every language. However, the phishing warning page is currently disabled completely, so it shouldn't really affect anything to leave it as-is.

Reply:
Okey, i suggest so "Ce site pourrait vous nuire", as it matches more the english translation. ("peut" is more like "can harm you") 
It's going to be committed